js/google_ima.js

```diff
@@ -2426,7 +2438,10 @@ require("../html5-common/js/utils/utils.js");
     this.mute = function() {
       //Note there is no "mute" API. IMA seems to handle the muted
       //attribute themselves when volume is set to 0
-      volumeWhenMuted = _ima.getVolume();
+      var currentVolume = _ima.getVolume();
+      if (currentVolume) {
+        volumeWhenMuted = currentVolume;
```

Comment:
Will this fail to set `volumeWhenMuted` when volume is 0?

Reply:
Yea, but this is deliberate again. We want to store the volume when we mute, but if the current volume is alerady 0, we don't want to store it. When we restore the volume, it will be whatever value we grab here, or if we didn't grab the value, we set it to 1 (see the other line you commented on)